diff --git a/CMakeLists.txt b/CMakeLists.txt
index ea8b330c..9d1d0fc0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -424,6 +424,7 @@ endif()
 # This preprocessor define will drive the build, also used in the *.pc files:
 if("${HWY_LIBRARY_TYPE}" STREQUAL "SHARED")
   set(DLLEXPORT_TO_DEFINE "HWY_SHARED_DEFINE")
+  list(APPEND HWY_FLAGS -Dhwy_EXPORTS)
 else()
   set(DLLEXPORT_TO_DEFINE "HWY_STATIC_DEFINE")
 endif()
