diff --git a/CMakeLists.txt b/CMakeLists.txt
index ea8b330c..9d1d0fc0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -424,6 +424,7 @@ endif()
 # This preprocessor define will drive the build, also used in the *.pc files:
 if("${HWY_LIBRARY_TYPE}" STREQUAL "SHARED")
   set(DLLEXPORT_TO_DEFINE "HWY_SHARED_DEFINE")
+  list(APPEND HWY_FLAGS -Dhwy_EXPORTS)
 else()
   set(DLLEXPORT_TO_DEFINE "HWY_STATIC_DEFINE")
 endif()
diff --git a/CMakeLists.txt.in b/CMakeLists.txt.in
index a0260b82..c71ef87c 100644
--- a/CMakeLists.txt.in
+++ b/CMakeLists.txt.in
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.10)
 
 project(googletest-download NONE)
 
